### PR TITLE
fix(tracking_object_merger): fix bug and rework parameters

### PR DESCRIPTION
**Pull request by kminoda**
## Description

Use param file as well as fixing a bug in param name.

Must be merged with https://github.com/autowarefoundation/autoware_launch/pull/823

## Tests performed

Launch Autoware and parameters are successfully loaded.

## Effects on system behavior

Now `logging_file_path` will be `/tmp/decorative_tracker_merger.log` and `enable_logging` will be `false` by default in Autoware (see https://github.com/autowarefoundation/autoware_launch/pull/823). 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]

**Comment by kminoda** [comment]
@YoshiRi Would appreciate if you could briefly check this PR :pray: 
This will fix a bug, which will change the default parameter of `tracking_object_merger`. Let me know if it is better to set the default value something else.

File: perception/tracking_object_merger/config/decorative_tracker_merger.param.yaml [hunk 1]
```diff
@@ -22,5 +22,5 @@
       max_dt: 1.0
 
     # logging settings
-    enable_logging: false
-    log_file_path: "/tmp/decorative_tracker_merger.log"
+    enable_logging: true
```
Comment (YoshiRi) [inline perception/tracking_object_merger/config/decorative_tracker_merger.param.yaml:25]: Thanks for reworking.
This packages logging information is too detailed, so I do not set this True now.

```suggestion
    enable_logging: false
```

**Review by YoshiRi — CHANGES_REQUESTED** [review]
Sorry, I forgot to change this logging settings. Could you fix it?

**Review by YoshiRi — APPROVED** [review]
LGTM

**Comment by kminoda** [comment]
@YoshiRi Would you also approve https://github.com/autowarefoundation/autoware_launch/pull/823?

**Pull request merged**